components/dashboard/src/settings/Preferences.tsx

```diff
@@ -118,50 +104,48 @@ export default function Preferences() {
     return <div>
         <PageWithSubMenu subMenu={settingsMenu} title='Preferences' subtitle='Configure user preferences.'>
             {ideOptions && browserIdeOptions && <>
-                <h3>Editor</h3>
-                <p className="text-base text-gray-500 dark:text-gray-400">Choose which IDE you want to use.</p>
-                <div className="my-4 space-x-4 flex">
-                    {
-                        browserIdeOptions.map(([id, option]) => {
-                            const selected = defaultIde === id;
-                            const onSelect = () => actuallySetDefaultIde(id);
-                            return renderIdeOption(option, selected, onSelect);
-                        })
-                    }
-                </div>
-                {ideOptions.options[defaultIde].notes &&
-                    <InfoBox className="my-5 max-w-2xl"><ul>
-                        {ideOptions.options[defaultIde].notes?.map((x, idx) => <li className={idx > 0 ? "mt-2" : ""}>{x}</li>)}
-                    </ul></InfoBox>
-                }
-                {desktopIdeOptions && desktopIdeOptions.length > 0 && <>
-                    <div className="mt-4 space-x-4 flex">
-                        <CheckBox
-                            title={<div>Open in Desktop IDE <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Beta</PillLabel></div>}
-                            desc="Choose whether you would like to open your workspace in a desktop IDE instead."
-                            checked={useDesktopIde}
-                            onChange={(evt) => actuallySetUseDesktopIde(evt.target.checked)} />
+                {browserIdeOptions && <>
```

Comment:
`browserIdeOptions` is already checked on the line above, no need to check a second time:

```suggestion
```